Teacher Workout/Features/Authentication/SignIn/ViewModel/SignInViewModel.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+class SignInViewModel: ObservableObject {
+    @Published var email: String = ""
+    @Published var password: String = ""
+    @Published var showEmailError: Bool = false
+    @Published var showPasswordError: Bool = false
+
+    let validator = Validator.shared
```

Comment:
We should use `private` for this property if it's not accessed from other classes.